subsystem/kubeapi_test.go

```diff
@@ -647,15 +648,68 @@ var _ = Describe("[kube-api]cluster installation", func() {
 
 		infraEnv := getInfraEnvCRD(ctx, kubeClient, infraEnvKubeName)
 		firstURL := infraEnv.Status.ISODownloadURL
+		firstCreatedAt := infraEnv.Status.CreatedAt
+
+		annotations := map[string]string{"foo": "bar"}
+		By("Update InfraEnv Annotations")
+		Eventually(func() error {
+			infraEnv.SetAnnotations(annotations)
+			return kubeClient.Update(ctx, infraEnv)
+		}, "30s", "10s").Should(BeNil())
+
+		Consistently(func() map[string]string {
+			return getInfraEnvCRD(ctx, kubeClient, infraEnvKubeName).GetAnnotations()
+		}, "30s", "2s").Should(Equal(annotations))
+
+		By("Verify InfraEnv Status ISODownloadURL has not changed")
+		Consistently(func() string {
+			return getInfraEnvCRD(ctx, kubeClient, infraEnvKubeName).Status.ISODownloadURL
+		}, "30s", "2s").Should(Equal(firstURL))
+
+		By("Verify InfraEnv Status CreatedAt has not changed")
+		Consistently(func() *metav1.Time {
```

Comment:
Consistently is not needed, it can be replaced with expect.
In this case you already verified taht the URL didn't change.